Python/symtable.c

```diff
@@ -558,6 +559,65 @@ analyze_name(PySTEntryObject *ste, PyObject *scopes, PyObject *name, long flags,
     return 1;
 }
 
+static int
+is_free_in_children(PySTEntryObject *entry, PyObject *key) {
```

Comment:
```suggestion
is_free_in_child(PySTEntryObject *entry, PyObject *key) {
```

(it checks for one, not all children)